packages/nextjs/app/builders/0x04cdfb463600a29e8e93053b88daa11f54379907/page.tsx

```diff
@@ -0,0 +1,467 @@
+                <SkillTag icon={<Code2 size={16} />} label="Solidity" isDark={isDark} />
+                <SkillTag icon={<Gem size={16} />} label="Ethereum" isDark={isDark} />
+                <SkillTag icon={<ScrollText size={16} />} label="Smart Contracts" isDark={isDark} />
+                <SkillTag icon={<Layers size={16} />} label="Arbitrum" isDark={isDark} />
+                <SkillTag icon={<Shield size={16} />} label="Security" isDark={isDark} />
+                <SkillTag icon={<Lock size={16} />} label="Cryptography" isDark={isDark} />
+                <SkillTag icon={<Palette size={16} />} label="Design" isDark={isDark} />
+                <SkillTag icon={<Globe size={16} />} label="Web3.js" isDark={isDark} />
+              </div>
+            </div>
+
+            <div className="w-full space-y-4">
+              <h3
+                className={`text-left text-xs font-bold uppercase tracking-widest pl-1 opacity-70 ${theme.mutedText}`}
+              >
+                Achievements
+              </h3>
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+                <AchievementCard
+                  title="Check-in Complete"
+                  desc="Successfully checked into BatchRegistry"
+                  active={true}
+                  isDark={isDark}
+                />
+                <AchievementCard
+                  title="Batch 22 Member"
+                  desc="Active participant in BuidlGuidl"
+                  icon={<Award className="text-yellow-500" size={20} />}
+                  isDark={isDark}
+                />
+              </div>
+            </div>
+
+            <div className={`w-full pt-6 border-t ${theme.divider}`}>
+              <h3
+                className={`text-left text-xs font-bold uppercase tracking-widest pl-1 mb-4 opacity-70 ${theme.mutedText}`}
+              >
+                Connect
+              </h3>
+
+              <div className="uiverse-card w-full flex-wrap justify-center sm:justify-start">
+                <a
+                  href="https://github.com/dharmanan"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="socialContainer containerGithub group"
+                  title="GitHub"
+                >
+                  <svg
+                    className="socialSvg"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                    strokeWidth="2"
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                  >
+                    <path d="M15 22v-4a4.8 4.8 0 0 0-1-3.5c3 0 6-2 6-5.5.08-1.25-.27-2.48-1-3.5.28-1.15.28-2.35 0-3.5 0 0-1 0 3-1.5-2.64-.5-5.36-.5-8 0C6 2 5 2 5 2c-.3 1.15-.3 2.35 0 3.5A5.403 5.403 0 0 0 4 9c0 3.5 3 5.5 6 5.5-.39.49-.68 1.05-.85 1.65-.17.6-.22 1.23-.15 1.85v4" />
+                    <path d="M9 18c-4.51 2-5-2-7-2" />
+                  </svg>
```

Comment:
I think these SVGs could be stored in a separate file each, since they take up too much space in the code.